Automattic/wp-calypso
# Unified design picker: Move legacy design removal logic to API

#### Proposed Changes

This PR deletes the logic that removes legacy design variations in favor of removing this logic to the API, thus simplifying the code on Calypso side. The list of legacy designs is as follows:

- "quadrat-red"
- "twentytwentytwo-mint"
- "geologist-slate"
- "quadrat-white"
- "twentytwentytwo-blue"
- "quadrat-green"
- "geologist-yellow"
- "geologist-blue"
- "twentytwentytwo-pink"
- "quadrat-yellow"
- "quadrat-black"
- "twentytwentytwo-swiss"
- "geologist-cream"
- "twentytwentytwo-red"

~~⚠️ Note that this PR should be merged AFTER D89334-code has been deployed.~~
✅ D89334-code has been deployed.

#### Testing Instructions



* Head to the unified design picker `/setup/designSetup?siteSlug=${site_slug}`
* Ensure that legacy design variations are showing up again.
* Ensure that the style selection experience works as before.

#### Pre-merge Checklist






Related to #
